src/Api/ProjectsApi.php

```diff
@@ -75,4 +129,58 @@ public function projectsUserUserIdGet(string $userId, string $maxVersion = null,
   {
     // TODO: Implement projectsUserUserIdGet() method.
   }
+
+  /**
+   * @param Program[] $programs
+   *
+   * @return Project[]
+   */
+  private function getResponseData($programs)
```

Comment:
features project, for example, have a different response. Use a different name for this method. 
E.g. getProjectsResponseData()